e2e-tests/monitoring-2-0/run

```diff
@@ -7,6 +7,25 @@ test_dir=$(realpath $(dirname $0))
 
 set_debug
 
+get_node_id_from_pmm() {
+	local -a nodeList=()
+	for instance in $(kubectl_bin get pods --no-headers -l app.kubernetes.io/component=pxc --output=custom-columns='NAME:.metadata.name'); do
+		nodeList+=($(kubectl_bin exec -n "$namespace" $instance -c pmm-client -- pmm-admin status --json | jq -r '.pmm_agent_status.node_id'))
+	done
+
+	echo "${nodeList[@]}"
+}
+
+is_node_id_exists() {
```

Comment:
maybe `check_node_id_exists` would be better or even `does_node_id_exists`?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1416/commits/b19ff569e811763cde0a5ba8a2c67fe6dac8ffde done